package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "css-loader",
-  "version": "0.26.2",
+  "version": "0.26.4",
```

Comment:
@ekulabuhov Good catch 💯 I totally missed it. 

In a short while, if seen this a few times now, maybe we should update the `PR_TEMPLATE` to explicitly mention that `npm version` is always handled/done by us? cc @bebraw @ d3viant0ne